CondCore/CondDB/interface/IOVSchema.h

```diff
@@ -10,6 +10,8 @@ namespace cond {
 
   namespace persistency {
 
+    cond::Hash makeHash(const std::string& objectType, const cond::Binary& data);
```

Comment:
@smuzaffar out of curiosity: what is the purpose of this new method defined here?

Reply:
https://github.com/cms-sw/cmssw/blob/79d9ef38a7392b5a532dc7522f2c12f50be03e1b/CondCore/Utilities/bin/conddb_test_gt_perf.cpp#L5 was including `CondCore/CondDB/src/IOVSchema.cc` to call `makeHash` ( https://github.com/cms-sw/cmssw/blob/79d9ef38a7392b5a532dc7522f2c12f50be03e1b/CondCore/Utilities/bin/conddb_test_gt_perf.cpp#L193). I moved the declaration in the header so that `CondCore/Utilities/bin/conddb_test_gt_perf.cpp` can include `CondCore/CondDB/interface/IOVSchema.h`